Grammar.html

```diff
@@ -17,45 +17,59 @@
 <li>Literals other than keywords are <font color="green">"</font><tt>quoted-monospaced</tt><font color="green">"</font> text.</li>
 <li>Non-terminals are set in <i>italics</i>, with <font color="#003399"><i>blue italics</i></font> used for links.</li>
 <li>EBNF meta-characters are <font color="green">green</font>.</li>
-</ul>
+</ul></p>
```

Comment:
The paragraph should be closed in line 14, i.e., before the unordered list. See https://stackoverflow.com/a/7168757/8520615 for some references.

Reply:
Fixed.